File: src/main/java/client/inventory/item/templates/PortalScrollItemTemplate.java
package client.inventory.item.templates;

import util.packet.PacketReader;

public class PortalScrollItemTemplate extends ItemBundleTemplate {

    private final int moveTo;

    public int getMoveTo() {
        return moveTo;
    }

    public PortalScrollItemTemplate(int id, PacketReader r) {
        super(id, r);
        moveTo = r.readInteger();
    }
}

File: src/main/java/client/inventory/item/templates/PortalScrollItemTemplate.kt
package client.inventory.item.templates

import util.packet.PacketReader

class PortalScrollItemTemplate(id: Int) : ItemBundleTemplate(id) {

    var moveTo: Int = 0
        private set

    override fun decode(r: PacketReader): PortalScrollItemTemplate {
        super.decode(r)

        moveTo = r.readInteger()

        return this
    }
}